client/trino-jdbc/pom.xml

```diff
@@ -79,6 +79,12 @@
             <groupId>io.trino</groupId>
             <artifactId>trino-hive-hadoop2</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
nit: commit message `Add dockerized hive minio data lake for tests` -> `Add dockerized Hive with Minio data lake for tests`